src/test/java/org/creekservice/kafka/test/perf/implementations/DevHarrelImplementationTest.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2023 Creek Contributors (https://github.com/creek-service)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.creekservice.kafka.test.perf.implementations;
+
+class DevHarrelImplementationTest extends ImplementationTest {}
```

Comment:
Class no longer needed (again, this is me moving the goal posts on you!)

Reply:
Hmm, how so? I can still see test classes for other implementations

Reply:
Sorry, I'm tripping - thought this was a different class.